### PR TITLE
ODIM-5405 ODIM created events the EventTimestamp,Message and Severity parameters are missing

### PR DESCRIPTION
**Pull request by irappa-pattar**

### CODE REVIEW

**Review by jeevan-kamkar — APPROVED** [review]

**Review by amar-shalgar — APPROVED** [review]

File: svc-aggregation/agmessagebus/publish.go [hunk 1]
```diff
@@ -35,12 +36,15 @@ func Publish(systemID, eventType, collectionType string) {
...
 
 	var event = common.Event{
-		EventID:   uuid.NewV4().String(),
-		MessageID: "ResourceEvent.1.0.3." + eventType,
-		EventType: eventType,
+		EventID:        uuid.NewV4().String(),
+		MessageID:      "ResourceEvent.1.0.3." + eventType,
+		EventTimestamp: time.Now().Format(time.RFC3339),
+		EventType:      eventType,
+		Message:        "Resource event",
```
Comment (shivacharanms) [inline svc-aggregation/agmessagebus/publish.go]: Update event message in accordance to what the event is about in aggregation service

File: svc-task/tmessagebus/publish.go [hunk 2]
```diff
@@ -36,12 +37,15 @@ func Publish(taskURI string, messageID string, eventType string) {
...
 	var eventID = uuid.NewV4().String()
 	var event = common.Event{
-		EventID:   eventID,
-		MessageID: messageID,
-		EventType: eventType,
+		EventID:        eventID,
+		MessageID:      messageID,
+		EventTimestamp: time.Now().Format(time.RFC3339),
+		EventType:      eventType,
+		Message:        "Task event",
```
Comment (shivacharanms) [inline svc-task/tmessagebus/publish.go]: Update event message in accordance to what the event is about in task service

**Review by shivacharanms — CHANGES_REQUESTED** [review]

**Review by shivacharanms — APPROVED** [review]

File: svc-aggregation/agmessagebus/publish.go (hunk 1 above)
Comment (irappa-pattar) [reply svc-aggregation/agmessagebus/publish.go]: updated

File: svc-task/tmessagebus/publish.go (hunk 2 above)
Comment (irappa-pattar) [reply svc-task/tmessagebus/publish.go]: updated

**Pull request merged**